code/script.js

```diff
@@ -26,22 +34,68 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Conversation starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hi there! Welcome to Movie-Bob, the friendly movie picking bot. What's your name?`, 'bot')
+}
+
+const ageQuestion = () => {
+  showMessage('Nice to meet you! How old are you?', 'bot')
+  currentQuestion =`question2`
+}
+
+const resetForm = () => {
+  location.reload();
+};
```

Comment:
I might be easier to understand if the variables that are not global, follow the conversation flow. Just so we don't break the flow with information that are not necessary at the time.